homeassistant/components/nightscout/translations/ca.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We don't add translations except the default English that is generated by the translations develop script.